src/session-pool.ts

```diff
@@ -990,6 +1044,10 @@ export class SessionPool extends EventEmitter implements SessionPoolInterface {
    * @returns {Promise}
    */
   async _ping(session: Session): Promise<void> {
+    // NOTE: Please do not trace Ping as it gets quite spammy
```

Comment:
How are we making sure this does not happen. ?

Reply:
Keeping that documentation in, then later on once we have traceWrap in, adding explicit tests that ping doesn't produce any spans.

Reply:
understood.